pandas/io/tests/parser/common.py

```diff
@@ -1635,3 +1635,12 @@ def test_file_handles(self):
                 if PY3:
                     self.assertFalse(m.closed)
                 m.close()
+
+    def test_file_system_encoding(self):
+        """
+        Test compatibility with file system encoding.
+        """
+        for filename in ('test_e.txt', 'test_é.txt'):
+            with tm.ensure_clean(filename) as path:
+                pd.DataFrame().to_csv(path)
+                pd.read_csv(path)
```

Comment:
use ``self.read_csv`` to read with the various parsers

use a non-zero dataframe, then compare the return results.